src/main/webapp/app/iris/settings/iris-settings-update/iris-common-sub-settings-update/iris-common-sub-settings-update.component.html

```diff
@@ -0,0 +1,64 @@
+<div class="form-check form-switch">
+    <input
+        class="form-check-input"
+        type="checkbox"
+        id="enabledDisabledSwitch"
+        [disabled]="!isAdmin && settingsType !== EXERCISE"
+        [(ngModel)]="enabled"
+        (change)="onEnabledChange()"
+    />
+    <label class="form-check-label" for="enabledDisabledSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.enabled-disabled"> Enabled/Disabled </label>
+</div>
+
+<h4 class="form-label mt-3" jhiTranslate="artemisApp.iris.settings.subSettings.models.title">Models</h4>
+<span><span class="fw-bold fs-5" jhiTranslate="artemisApp.iris.settings.subSettings.models.allowedModels.title">Allowed Models</span>: </span>
+<div *ngIf="parentSubSettings" class="form-check form-switch">
+    <input
+        class="form-check-input"
+        type="checkbox"
+        id="inheritAllowedModelsSwitch"
+        [disabled]="!isAdmin"
+        [(ngModel)]="inheritAllowedModels"
+        (change)="onInheritAllowedModelsChange()"
+    />
+    <label class="form-check-label" for="inheritAllowedModelsSwitch" jhiTranslate="artemisApp.iris.settings.subSettings.models.allowedModels.inheritSwitch"> Inherit </label>
+</div>
+<div class="form-group">
+    <div class="form-check" *ngFor="let model of allIrisModels">
+        <input
+            class="form-check-input"
+            type="checkbox"
+            id="{{ model.id }}"
+            [disabled]="!isAdmin || inheritAllowedModels"
+            [ngModel]="allowedIrisModels.includes(model)"
+            (ngModelChange)="onAllowedIrisModelsSelectionChange(model)"
+        />
+        <label class="form-check-label" for="{{ model.id }}">
+            {{ model.name }}
+        </label>
+    </div>
+</div>
+
+<h5 class="mt-3"><span class="fw-bold" jhiTranslate="artemisApp.iris.settings.subSettings.models.preferredModel.title">Preferred Model</span>:</h5>
+<div class="d-flex align-items-center">
+    <div ngbDropdown class="d-inline-block">
+        <button class="btn btn-outline-primary w-100" id="dropdownBasic1" ngbDropdownToggle>
+            {{ getPreferredModelName() ?? ('artemisApp.iris.settings.subSettings.models.preferredModel.inherit' | artemisTranslate) }}
```

Comment:
The fallback value has problem when allowedModel is empty.
<img width="1440" alt="Screenshot 2023-11-12 at 20 20 49" src="https://github.com/ls1intum/Artemis/assets/60965100/dbbab364-620e-428e-b136-b0828fe5ac70">

Reply:
This is a unique edge case for the global settings, as it can't inherit from anywhere, so it can't be selected here.